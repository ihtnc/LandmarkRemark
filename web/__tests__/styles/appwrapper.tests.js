import React from 'react';
import { shallow } from 'enzyme';
import AppWrapper from "@styles/appwrapper";

describe("AppWrapper styled component", () => {
  test("should match latest snapshot", () => {
    const wrapper = shallow(<AppWrapper />)
    expect(wrapper).toMatchSnapshot();
  });
});