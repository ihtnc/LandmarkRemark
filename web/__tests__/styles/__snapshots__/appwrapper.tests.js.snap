// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppWrapper styled component should match latest snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  width: 100%;
  height: 100%;
",
        ],
      },
      "displayName": "styled.section",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "section",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
